### PR TITLE
Update code example in Federated ETL Backup doc

### PR DESCRIPTION
**Pull request by thomasvn**
* Update the code block to specify that this is just an abridged example to provide context. The full example in the link is required for proper configuration.
* Also mention configuration for `object-store.yaml` and `cluster_id`.

### CODE REVIEW

**Comment by thomasvn** [comment]
@jessegoodier Thoughts? I know you've been testing the minimum viable Thanos sidecar configuration.

My approach here, is that I'd prefer to maintain and update the thanos-sidecar configuration in `values-thanos.yaml` rather than this document. But I still want to leave a small code block in the doc because it provides good context as to what the config change would look like.

File: federated-etl-backups-alerting.md [hunk 1]
```diff
@@ -19,21 +19,22 @@ prometheus:
 For long term storage of Prometheus metrics, we recommend setting up a Thanos sidecar container to push Prometheus metrics to a cloud storage bucket.
 
 ```yaml
+# This is an abridged example. Full example in link below.
 prometheus:
-  extraArgs:
```
Comment (jessegoodier) [inline federated-etl-backups-alerting.md:23]: these are part of the minimum requirements.

File: federated-etl-backups-alerting.md [hunk 2]
```diff
@@ -19,21 +19,22 @@ prometheus:
...
-    storage.tsdb.min-block-duration: 2h
-    storage.tsdb.max-block-duration: 2h
   server:
     sidecarContainers:
     - name: thanos-sidecar
-      image: thanosio/thanos:latest
+      image: thanosio/thanos:v0.29.0
```
Comment (jessegoodier) [inline federated-etl-backups-alerting.md:27]: I'm using
thanosio/thanos:v0.30.2
works great.

**Comment by bstuder99** [comment]
I will go in and make edits + add it to the style guide, but let's format thanos-sidecar as Thanos sidecar, as seen in Thanos' docs: https://thanos.io/tip/components/sidecar.md/

**Review by bstuder99 — DISMISSED** [review]

**Comment by thomasvn** [comment]
@bstuder99 @jessegoodier . I had a side conversation with Jesse, and we agreed to put the more complete code block in the docs as an example. Requesting another review!

**Pull request merged**